__tests__/components/adminTable.test.jsx

```diff
@@ -0,0 +1,52 @@
+import AdminTable from '../../components/adminTable';
+import React from 'react';
+import renderer from 'react-test-renderer';
+const sampleColumns = [
+    {
+      name: 'Name',
+      selector: row => row.name
+    },
+    {
+      name: 'Email',
+      selector: row => row.userEmail
+    },
+    {
+      name: 'Role',
+      selector: row => row.role
+    },
+    {
+      name: 'Actions',
+      selector: row => row.adminActions
+    }
+  ];
+const sampleUsers=[
+    {
+        id: 1,
+        name: "Hamzat Victor",
+        email: "oluwaborihamzat@gmail.com",
+        role: "ADMIN"
+    },
+    {
+        id: 2,
+        name: "Alade Christopher",
+        email: "aladechristoph@gmail.com",
+        role: "TEACHER"
+    },
+    {
+        id: 3,
+        name: "Ayomide onifade",
+        email: "Jangulabi@gmail.com",
+        role: "TEACHER"
+    },
+]
+
+describe('AdminTable rendering correctly', () => {
+  it('renders correctly', () => {
```

Comment:
Can you write a more descriptive name for this test, something more specific than "renders correctly"?  For example, "renders X rows and Y columns"

Reply:
Noted, I'll make some changes.

Reply:
I just updated the test name and made it more descriptive.